client/src/Filters/TravelType.jsx

```diff
@@ -4,11 +4,50 @@ class TravelType extends React.Component {
 
   constructor(props) {
     super(props);
+
+    this.state = {
+      isToggleOn: false
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+
+    event.preventDefault();
+
+    var setToggle = () => {
```

Comment:
I was trying to think of a way you could abstract the setToggle function since it seems like many of your components need it, but I'm honestly not sure how since it needs to uniquely update each indv component's state.. just something to think about